Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/WheelWithVariableSpeed.mo

```diff
@@ -0,0 +1,227 @@
+    "Nominal latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsSenCooPL(final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsLatCooPL(final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsSenHea_nominal(final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsLatHea_nominal(final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsSenHeaPL(final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the heating mode";
+  parameter Modelica.Units.SI.Efficiency epsLatHeaPL(final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the heating mode";
+  Modelica.Blocks.Interfaces.RealInput wheSpe(final unit="1")
+    "Wheel speed ratio"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
+  Modelica.Blocks.Interfaces.RealOutput P
+    "Electric power consumed by the wheel"
+    annotation (Placement(transformation(extent={{100,-10},{120,10}})));
+  Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.BaseClasses.HeatExchangerWithInputEffectiveness
+    hex(
+    redeclare package Medium1 = Medium1,
+    redeclare package Medium2 = Medium2,
+    final m1_flow_nominal=m1_flow_nominal,
+    final m2_flow_nominal=m2_flow_nominal,
+    final show_T=true,
+    final dp1_nominal=dp1_nominal,
+    final dp2_nominal=dp2_nominal)
+    "Heat exchanger"
+    annotation (Placement(transformation(extent={{6,-10},{26,10}})));
+  Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.BaseClasses.Effectiveness
+    effCal(
+    final epsSenCoo_nominal=epsSenCoo_nominal,
+    final epsLatCoo_nominal=epsLatCoo_nominal,
+    final epsSenCooPL=epsSenCooPL,
+    final epsLatCooPL=epsLatCooPL,
+    final epsSenHea_nominal=epsSenHea_nominal,
+    final epsLatHea_nominal=epsLatHea_nominal,
+    final epsSenHeaPL=epsSenHeaPL,
+    final epsLatHeaPL=epsLatHeaPL,
+    final VSup_flow_nominal=m1_flow_nominal/1.293)
+    "Calculates the effectiveness of heat exchange"
+    annotation (Placement(transformation(extent={{-40,40},{-20,60}})));
+  Buildings.Fluid.Sensors.VolumeFlowRate senExhFlow(
+    redeclare package Medium = Medium2,
+    final m_flow_nominal=m2_flow_nominal)
+    "Flow sensor in the exhaust air stream"
+    annotation (Placement(transformation(extent={{56,-14},{42,2}})));
+  Buildings.Fluid.Sensors.VolumeFlowRate senSupFlow(
+    redeclare package Medium =Medium1,
+    final m_flow_nominal=m1_flow_nominal)
+    "Flow sensor in the supply air stream"
+    annotation (Placement(transformation(extent={{-42,92},{-28,108}})));
+  Modelica.Blocks.Sources.RealExpression TSup(final y=Medium1.temperature(
```

Comment:
Can we use the temperature sensor, rather than using the real expression which is not visually clear as the connection? You may also change the exhaust air temperature measurement.

Please also change it in other model.